doc/getting_started/index.rst

```diff
@@ -471,71 +471,142 @@ Install a Toolchain
 
-            * ``$HOME/zephyr-sdk[-x.y.z]``
-            * ``$HOME/.local/zephyr-sdk[-x.y.z]``
-            * ``$HOME/.local/opt/zephyr-sdk[-x.y.z]``
-            * ``$HOME/bin/zephyr-sdk[-x.y.z]``
-            * ``/opt/zephyr-sdk[-x.y.z]``
-            * ``/usr/zephyr-sdk[-x.y.z]``
-            * ``/usr/local/zephyr-sdk[-x.y.z]``
+      #. Run the Zephyr SDK bundle setup script:
 
-            where ``[-x.y.z]`` is optional text, and can be any text, for example ``-0.13.2``.
+         .. code-block:: bash
+
+            cd zephyr-sdk-0.14.0
+            ./setup.sh
 
-            If installing the Zephyr SDK outside any of those locations, please read: :ref:`zephyr_sdk`
+         .. note::
+            You only need to run the setup script once after extracting the Zephyr SDK bundle.
 
-            You cannot move the SDK directory after you have installed it.
+            It may be necessary to re-run the setup script if you relocate the Zephyr SDK bundle
+            directory after the initial setup.
 
       #. Install `udev <https://en.wikipedia.org/wiki/Udev>`_ rules, which
          allow you to flash most Zephyr boards as a regular user:
 
          .. code-block:: bash
 
-            sudo cp ~/zephyr-sdk-0.13.2/sysroots/x86_64-pokysdk-linux/usr/share/openocd/contrib/60-openocd.rules /etc/udev/rules.d
+            sudo cp ~/zephyr-sdk-0.14.0/sysroots/x86_64-pokysdk-linux/usr/share/openocd/contrib/60-openocd.rules /etc/udev/rules.d
             sudo udevadm control --reload
 
    .. group-tab:: macOS
 
-      Follow the instructions in :ref:`gs_toolchain`. Note that the Zephyr SDK
-      is not available on macOS.
+      #. Download and verify the `latest Zephyr SDK bundle
+         <https://github.com/zephyrproject-rtos/sdk-ng/releases>`_:
+
+         .. code-block:: bash
+
+            cd ~
+            wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.14.0/zephyr-sdk-0.14.0_macos-x86_64.tar.gz
+            wget -O - https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.14.0/sha256.sum | shasum --check --ignore-missing
+
+      #. Extract the Zephyr SDK bundle archive:
+
+         .. code-block:: bash
+
+            tar xvf zephyr-sdk-0.14.0_macos-x86_64.tar.gz
+
+         .. note::
+            It is recommended to extract the Zephyr SDK bundle at one of the following locations:
+
+            * ``$HOME``
+            * ``$HOME/bin``
+            * ``/opt``
+            * ``/usr/local``
+
+            The Zephyr SDK bundle archive contains the ``zephyr-sdk-0.14.0`` directory and, when
+            extracted under ``$HOME``, the resulting installation path will be
+            ``$HOME/zephyr-sdk-0.14.0``.
+
+      #. Run the Zephyr SDK bundle setup script:
 
-      Do not forget to set the required :ref:`environment variables <env_vars>`
-      (:envvar:`ZEPHYR_TOOLCHAIN_VARIANT` and toolchain specific ones).
+         .. code-block:: bash
+
+            cd zephyr-sdk-0.14.0
+            ./setup.sh
```

Comment:
this works on macOS?

Reply:
Yes.

https://github.com/zephyrproject-rtos/sdk-ng/blob/6e7d5d6d7ff0cc774d77c50a3c32e3ad2be73ba2/scripts/template_setup_posix#L1

AFAIK, bash is included in all macOS distributions, although they might not be up-to-date.